documentation/release_notes.rst

```diff
@@ -8,6 +8,117 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.7.0
+===============
+
+New Features
+------------
+- Improved performance of the ``adjacent_find``, ``all_of``, ``any_of``, ``copy_if``, ``exclusive_scan``, ``equal``,
+  ``find``, ``find_if``, ``find_end``, ``find_first_of``, ``find_if_not``, ``inclusive_scan``, ``includes``,
+  ``is_heap``, ``is_heap_until``, ``is_partitioned``, ``is_sorted``, ``is_sorted_until``, ``lexicographical_compare``,
+  ``max_element``, ``min_element``, ``minmax_element``, ``mismatch``, ``none_of``, ``partition``, ``partition_copy``,
+  ``reduce``, ``remove``, ``remove_copy``, ``remove_copy_if``, ``remove_if``, ``search``, ``search_n``,
+  ``stable_partition``, ``transform_exclusive_scan``, ``transform_inclusive_scan``, ``unique``, and ``unique_copy``
+  algorithms with device policies. 
+- Improved performance of ``sort``, ``stable_sort`` and ``sort_by_key`` algorithms with device policies when using Merge
+  sort [#fnote1]_.
+- Added ``stable_sort_by_key`` algorithm in ``namespace oneapi::dpl``. 
+- Added parallel range algorithms in ``namespace oneapi::dpl::ranges``: ``all_of``, ``any_of``,
+  ``none_of``, ``for_each``, ``find``, ``find_if``, ``find_if_not``, ``adjacent_find``, ``search``, ``search_n``,
+  ``transform``, ``sort``, ``stable_sort``, ``is_sorted``, ``merge``, ``count``, ``count_if``, ``equal``, ``copy``,
+  ``copy_if``, ``min_element``, ``max_element``. These algorithms operate with C++20 random access ranges
+  and views while also taking an execution policy similarly to other oneDPL algorithms.
+- Added support for operators ==, !=, << and >> for RNG engines and distributions.
+- Added experimental support for the Philox RNG engine in ``namespace oneapi::dpl::experimental``.
+- Added the ``<oneapi/dpl/version>`` header containing oneDPL version macros and new feature testing macros.
+Fixed Issues
```

Comment:
Let's add one line to be aligned with the previous notes. Probably, it will not affect the page generated in html format, but it should be safer:

```suggestion

Fixed Issues
```